crates/forge/bin/cmd/create.rs

```diff
@@ -158,10 +158,28 @@ impl CreateArgs {
             let ZkContractArtifact { bytecode, hash, factory_dependencies, abi, .. } = artifact;
 
             let abi = abi.expect("Abi not found");
-
             let bin = bytecode.expect("Bytecode not found");
+
+            let bytecode = match bin.object() {
+                BytecodeObject::Bytecode(bytes) => bytes.to_vec(),
+                _ => {
+                    let link_refs = bin
```

Comment:
Still, I would like us to stop using the term *link references* to distinguish our *EraVM unlinked libraries* from *numeric* offsets used by *solc* for linking *EVM libraries*.

Reply:
Yeah it's literally to be compatible with upstream. We also still need to find a way to process linking the way zksolc wants anyways, since the artifact is lost by the time linking is done.
This is just so foundry detects the contract as unlinked and doesn't let you deploy it, but with #723 I plan to take care of clarifying this